docs/mocserver/mocserver.rst

```diff
@@ -11,433 +9,383 @@ Getting started
 
-MOC is an `IVOA standard`_ enabling description of arbitrary sky regions. Based on the HEALPix sky tessellation, it maps
-regions on the sky into hierarchically grouped predefined cells. It corresponds to a set of HEALPix cells at different
-orders.
+What's a MOC?
+-------------
 
-For those wanting to know more about MOCs, please refer to this `IVOA paper
-<http://ivoa.net/documents/MOC/20140602/REC-MOC-1.0-20140602.pdf>`_ and the `MOCPy's documentation
-<https://github.com/cds-astro/mocpy>`_ developed by the CDS.
+MOC means Multi-Order Coverage. It's an `IVOA standard`_ that allows to describe 
+Space-Time coverages of arbitrary sky regions --with or without
+an associated time of observation.
 
-CDS has set up a server known as the `MOCServer <http://alasky.unistra.fr/MocServer/query?>`_ storing data-set names
-each associated with a MOC spatial coverage and some meta-datas giving a more detailed explanation of the data-set.
+The space component maps the sky with the HEALPix sky
+tessellation to represent regions on the sky by hierarchically grouped HEALPix cells. 
+It other words, a Spatial MOC is a set of HEALPix cells at different orders.
 
-The MOCServer aims at returning the data-sets having at least one source lying in a specific sky region defined by the
-user. Internally the MOCServer performs the intersection between the given sky region and the MOCs associated with each
-data-sets. Because the MOC associated to a data-set describes its sky coverage, if the above intersection is not null
-then the MOCServer knows that some sources of this data-set are in the user defined sky region.
+The idea is the same for Time MOCs. The time axis is split into time cells.
 
-To be aware of what the MOCServer returns, please refers to this `link
-<http://alasky.unistra.fr/MocServer/query?RA=10.8&DEC=32.2&SR=1.5&intersect=overlaps&get=record&fmt=html>`_.
-We have queried the MOCServer with a cone region of center ra, dec = (10.8, 32.2) deg and radius = 1.5 deg. In return,
-the MOCServer gives a list of data-sets each tagged with an unique ID along with some other meta-datas too e.g.
-``obs_title``, ``obs_description``, ``moc_access_url`` (url for accessing the MOC associated with the data-set. Usually
-a FITS file storing a list of HEALPix cells).
+For those wanting to know more about MOCs, please refer to `the MOC 2.0 specification 
+document <https://ivoa.net/documents/MOC/20220727/REC-moc-2.0-20220727.pdf>`_.
 
-It is also possible to ask the MOCServer for retrieving data-sets based on their meta-data values. `Here
-<http://alasky.unistra.fr/MocServer/query?RA=10.8&DEC=32.2&SR=1.5&intersect=overlaps&get=record&fmt=html&expr=(dataproduct_type=image)>`_
-we have queried the MOCServer for only the image data-sets being in the cone defined above (``dataproduct_type``
-meta-data equals to ``"image"``).
+MOCPy is a Python library allowing to manipulate these Space-Time Coverage objects.
```

Comment:
Link the first mention of MOCPy to the github repo, and the documentation below to the rendered documentation.